lib/matplotlib/axes/_axes.py

```diff
@@ -6061,6 +6061,46 @@ def hist(self, x, bins=None, range=None, density=None, weights=None,
         bin_range = range
         del range
 
+        def _normalize_input(inp, ename='input'):
```

Comment:
I think the problem is that this function has disappeared on the master branch, so I think another approach to the problem is needed.